site2/docs/client-libraries-node.md

```diff
@@ -125,9 +128,11 @@ Pulsar Node.js producers have the following methods available:
 
 | Method | Description | Return type |
 | :----- | :---------- | :---------- |
-| `send(Object)` | Publishes a [message](#messages) to the producer's topic. When the message is successfully acknowledged by the Pulsar broker, or an error will be thrown, the Promise object run executor function. | `Promise<null>` |
-| `flush()` | Sends message from send queue to Pulser broker. When the message is successfully acknowledged by the Pulsar broker, or an error will be thrown, the Promise object run executor function. | `Promise<null>` |
-| `close()` | Closes the producer and releases all resources allocated to it. If `close()` is called then no more messages will be accepted from the publisher. This method will return Promise object, and when all pending publish requests have been persisted by Pulsar then run executor function. If an error is thrown, no pending writes will be retried. | `Promise<null>` |
+| `send(Object)` | Publishes a [message](#messages) to the producer's topic. When the message is successfully acknowledged by the Pulsar broker, or an error will be thrown, the Promise object whose result is the message ID runs executor function. | `Promise<Object>` |
```

Comment:
```suggestion
| `send(Object)` | Publishes a [message](#messages) to the producer's topic. When the message is successfully acknowledged by the Pulsar broker, or an error will be thrown, the Promise object whose result is the message ID runs executor function. | `Promise<Object>` |  
```
For the latter sentence, do you mean "When the message is successfully acknowledged by the Pulsar broker or an error is thrown, the Promise object whose result is the message ID runs executor function. 

Use present tense if you are covering facts that were, are, and forever shall be true. Always use the present tense in technical writing by default.

Reply:
fixed this. https://github.com/apache/pulsar/pull/10045/commits/70bcc41a5d79e5b2f19644eb9fd0aeed765234f8